Gopkg.toml

```diff
@@ -55,3 +55,14 @@
 [[constraint]]
   name = "github.com/writeas/go-strip-markdown"
   version = "2.0.0"
+
+# resolved to 1.1.3 which doesn't support flags used by apimachinary release-1.11
```

Comment:
these were set a while ago, did you try removing them and resolving to see if it built?